### ISSUE TITLE
Improve UX when providing command that's not supported

### ISSUE THREAD

**Issue by per-samuelsson** [Starcounter/Starcounter.Nova.Cli#52]
In `src\Starcounter.Cli`, do `dotnet run foo`. You'll see:

> Unhandled Exception: McMaster.Extensions.CommandLineUtils.CommandParsingException: Unrecognized command or argument 'foo'
>    at McMaster.Extensions.CommandLineUtils.CommandLineProcessor.HandleUnexpectedArg(String argTypeName) in C:\projects\commandlineutils\src\CommandLineUtils\Internal\CommandLineProcessor.cs:line 246
>    at McMaster.Extensions.CommandLineUtils.CommandLineProcessor.ProcessCommandOrArgument() in C:\projects\commandlineutils\src\CommandLineUtils\Internal\CommandLineProcessor.cs:line 135
>    at McMaster.Extensions.CommandLineUtils.CommandLineProcessor.ProcessNext() in C:\projects\commandlineutils\src\CommandLineUtils\Internal\CommandLineProcessor.cs:line 74
>    at McMaster.Extensions.CommandLineUtils.CommandLineProcessor.Process() in C:\projects\commandlineutils\src\CommandLineUtils\Internal\CommandLineProcessor.cs:line 35
>    at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Parse(String[] args) in C:\projects\commandlineutils\src\CommandLineUtils\CommandLineApplication.cs:line 532
>    at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args) in C:\projects\commandlineutils\src\CommandLineUtils\CommandLineApplication.cs:line 597
>    at Starcounter.Nova.Cli.Program.Main(String[] args) in C:\Users\Per\Git\Starcounter\Starcounter.Nova.Cli\src\Starcounter.Nova.Cli\Program.cs:line 51
>

**Comment by Mackiovello** [comment]
Good point. We should catch that exception and return a sensible error message instead.
